libc-bottom-half/cloudlibc/src/libc/sys/socket/accept.c

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Are files like this one derived from cloudlibc, or are they new? We should ideally put new code somewhere outside of the cloudlibc directory, to keep it clear when we're using and modifying upstream code which may have upstream assumptions, and when we're writing our own code.